Resources/Private/Fusion/Content/ContactForm.fusion

```diff
@@ -0,0 +1,73 @@
+##
+# "ContactForm" element
+#
+prototype(Neos.Demo:Content.ContactForm) < prototype(Neos.Neos:ContentComponent) {
+
+    renderer = Neos.Fusion.Form:Runtime.RuntimeForm {
+        namespace = 'contact'
+        process {
+            content = afx`
+                <fieldset>
+                    <legend>Contact</legend>
+                    <Neos.Fusion.Form:FieldContainer label="Name" field.name="name" attributes.class="form-group clearfix">
+                        <div class="input">
+                            <Neos.Fusion.Form:Input attributes.class="form-control" />
+                        </div>
+                    </Neos.Fusion.Form:FieldContainer>
+                    <Neos.Fusion.Form:FieldContainer label="Email" field.name="email" attributes.class="form-group clearfix">
+                        <div class="input">
+                            <Neos.Fusion.Form:Input attributes.class="form-control" />
+                        </div>
+                    </Neos.Fusion.Form:FieldContainer>
+                    <Neos.Fusion.Form:FieldContainer label="Message" field.name="message" attributes.class="form-group clearfix">
+                        <div class="input">
+                            <Neos.Fusion.Form:Textarea attributes.class="form-control" />
+                        </div>
+                    </Neos.Fusion.Form:FieldContainer>
+                </fieldset>
+            `
+
+            footer = afx`
+                <button type="submit" class="btn btn-primary">Submit</button>
+            `
+
+            schema {
+                name = ${Form.Schema.string().isRequired()}
+                email = ${Form.Schema.string().isRequired().validator('EmailAddress')}
+                message = ${Form.Schema.string().isRequired()}
+            }
+        }
+
+        action {
+            message {
```

Comment:
This one would have no effect if the redirect is enabled right?

Reply:
Yes, the message will be in the result but the location header will cause an immediate jump.